src/GraphQLParser/LexerContext.cs

```diff
@@ -8,27 +8,25 @@ namespace GraphQLParser
     internal struct LexerContext
```

Comment:
```suggestion
    internal ref struct LexerContext
```

Reply:
I have already tried this. It's getting slower 😕

Reply:
Doesn't this result in exactly the same compiled code, just placing additional restrictions on use?  Is it actually slower to add the ref in here?

Reply:
I'm just saying what I tried and it turned out to be slower. I've tried different approaches. You can experiment yourself, benchmarks are already prepared.

Reply:
lol - maybe if I have time I will just because I'm curious.  But I'm very happy going forward based on your findings.  I think we just need to wrap up some of these PRs so we can keep making progress.